src/applications/check-in/components/ErrorMessage.jsx

```diff
@@ -1,19 +1,14 @@
-import React, { useEffect } from 'react';
-import { focusElement } from 'platform/utilities/ui';
+import React from 'react';
 import PropTypes from 'prop-types';
 import { useTranslation } from 'react-i18next';
 
-const ErrorMessage = ({ header, message, additionalDetails }) => {
+const ErrorMessage = ({ message, additionalDetails }) => {
   const { t } = useTranslation();
-  const errorHeader = header ?? t('we-couldnt-check-you-in');
```

Comment:
To make this work with the wrapper component, I had to lift the header up into the Error component.

Is there a reason for keeping these two components separate? The only place ErrorMessage seems to be used independently is ErrorBoundary and I can't tell if that's actually in use. (Also a confirmation as to whether this change will break that or not if it *is* in use would be great!)